### ISSUE TITLE
Priority 1 accessibility issue: Visual styling section labels

### ISSUE THREAD

**Issue by alicerichmond** [wellcomecollection/wellcomecollection.org#6556]
**What**

Sections of the text have distinct purposes which are conveyed by visual styling. A bold font indicates that text acts as a section label. However, this distinction is not programmatically determinable.

<img width="523" alt="Screenshot 2021-05-27 at 15 17 22" src="https://user-images.githubusercontent.com/7315126/119842507-a4c0c700-befe-11eb-8382-952a79780720.png">

**Solution:**
Where visual styling is used to provide additional context or convey the purpose of information, please ensure that this context is programmatically determinable via the appropriate semantic mark-up. Screen reading software is reliant on correct semantic mark- up to determine structure and associations within the page in order to present this full context to users. For more information, please refer to F2: Failure of Success Criterion 1.3.1 due to using changes in text presentation to convey information without using the appropriate markup or text. In this case, a description list would be appropriate.

**Current code ref:**

<img width="523" alt="Screenshot 2021-05-27 at 15 17 51" src="https://user-images.githubusercontent.com/7315126/119842579-b5713d00-befe-11eb-8ed1-b75bd72e6c38.png">

**Where**

URL: https://wellcomecollection.org/works/hz43r7re
Page title: Francis Crick (1916-2004): archives | Wellcome Collection

**Comment by gestchild** [comment]
N.B. This html comes from the catalogue api

**Comment by pollecuttn** [comment]
Next step: Gareth to talk to Toni Hardy

**Comment by jennpb** [comment]
Not possible to create semantic headings based on the source data. This will need to be caveated in an accessibility statement for the site.

No further action.
